cmd/minikube/cmd/node_add.go

```diff
@@ -47,16 +47,23 @@ var nodeAddCmd = &cobra.Command{
 		if nodeName == "" {
 			name = profile + strconv.Itoa(len(mc.Nodes)+1)
```

Comment:
while we are at it, why not rename all mc to cc and make sure we call it cc everywhere in the code